README.md

```diff
@@ -344,12 +347,40 @@ Tapioca also supports pulling annotations from multiple sources:
 $ bin/tapioca annotations --sources https://raw.githubusercontent.com/$USER/$REPO1/$BRANCH https://raw.githubusercontent.com/$USER/$REPO2/$BRANCH
 ```
 
+#### Basic authentification
+
 Private repositories can be used as sources by passing the option `--auth` with an authentification string. For Github, this string is `token $TOKEN` where `$TOKEN` is a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token):
 
 ```shell
 $ bin/tapioca annotations --sources https://raw.githubusercontent.com/$USER/$PRIVATE_REPO/$BRANCH --auth "token $TOKEN"
 ```
 
+#### Using a .netrc file
+
+Tapioca supports reading credentials from a [netrc](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) file (defaulting to `~/.netrc`).
+
+Given these lines in your netrc:
+
+```netrc
+machine raw.githubusercontent.com
+  login $USERNAME
+  password $TOKEN
+```
+
+Where `$USERNAME` is your Github username and `$TOKEN` is a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). Then run Tapioca with the `--netrc` option (enabled by default).
+
+The `--netrc-file` option can be specified to read another file than `~/.netrc`:
+
+```shell
+$ bin/tapioca annotations --netrc --netrc-file /path/to/my/netrc/file
+```
+
+If the `--netrc-file` isn't specified, Tapioca will first try to find the `netrc` file from the environment by reading the `TAPIOCA_NETRC_FILE` variable then from `~/.netrc`:
+
+```shell
+$ TAPIOCA_NETRC_FILE=/path/to/my/netrc/file bin/tapioca annotations
+```
+
```

Comment:
Tapioca will first try to find the `netrc` file as specified by the `--netrc-file` option. If that option is not supplied, it will try the `TAPIOCA_NETRC_FILE` environment variable value. If that value is not supplied either, it will fallback to `~/.netrc`.